_static/css/furo-extensions.css

```diff
@@ -0,0 +1,225 @@
+#furo-sidebar-ad-placement {
```

Comment:
```suggestion
/*!
This file comes from `furo` package
Copyright (c) 2020 Pradyun Gedam <mail@pradyunsg.me>
SPDX-License-Identifier: MIT
*/
#furo-sidebar-ad-placement {
```

Reply:
I think that the best way would be creating a separate directory called `furo` in `css` directory and then within it paste `furo.css` and `furo-extensions.css` plus the following `LICENSE.MIT` file:
```
Copyright (c) 2020 Pradyun Gedam <mail@pradyunsg.me>

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to
deal in the Software without restriction, including without limitation the
rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
sell copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
IN THE SOFTWARE.
```